Source/WebKit/UIProcess/Inspector/win/RemoteWebInspectorUIProxyWin.cpp

```diff
@@ -131,9 +134,48 @@ WebPageProxy* RemoteWebInspectorUIProxy::platformCreateFrontendPageAndWindow()
     return m_webView->page();
 }
 
+void RemoteWebInspectorUIProxy::platformSave(Vector<WebCore::InspectorFrontendClient::SaveData>&& saveDatas, bool forceSaveAs)
+{
+    ASSERT(saveDatas.size() == 1);
+    UNUSED_PARAM(forceSaveAs);
+
+    // Remove custom URI schemes such as "web-inspector://" and also remove the leading "/".
+    URL url { saveDatas[0].url };
+    Vector<WCHAR, MAX_PATH> filePath(url.path().substring(1).toString().wideCharacters());
```

Comment:
This is not a right code even though it doesn't crash. The second template argument `MAX_PATH` is a capacity, not the vector size. You should explicitly allocate.

> Vector<WCHAR> filePath(MAX_PATH);
> ...

or

> auto filePath = url.path().substring(1).toString().wideCharacters();
> filePath.grow(MAX_PATH);
> ...

Reply:
It seems that the `m_inlineBuffer` inside the WTF::Vector is allocating the capacity size, so it did not crash.
However, it is not a good code as you said, so I will fix it.

Reply:
Ditto.
I used `filePath.grow`.